### PR TITLE
Remove use of hardcoded extents for displaying layers

### LINKED ISSUE #331: Remove the use of hardcoded bounds or zoom values for filtering layers
Display layers that do not have extents properties in their metadata globally.

### PR DESCRIPTION
**Pull request by crisner**
Fixes #331  (<=== Add issue number here)

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] PR is descriptively titled 📑 and links the original issue above 🔗
* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR
* [x] code is in uniquely-named feature branch and has no merge conflicts 📁
* [ ] screenshots/GIFs are attached 📎 in case of UI updation
* [ ] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software 

Thanks!

### CODE REVIEW

File: src/layerData.json [hunk 1]
```diff
@@ -7,15 +7,7 @@
...
-    "icon": "#096",
-    "extents": {
-      "bounds": [
-                  [-44.087585028245165, -148.88671875000003],
-                  [76.63922560965888, 140.62500000000003]
-                ],
-      "minZoom": 3,
-      "maxZoom": 15
-    }
+    "icon": "#096"
```
Comment (jywarren) [inline src/layerData.json]: OK great! Did you manually check these to ensure they should work outside the previously created bounds? Forgive me, i forget - did we figure out how these bounds were originally established? They're huge!

**Comment by jywarren** [comment]
Otherwise is this good to merge? Thank you!!!

File: src/layerData.json (hunk 1 above)
Comment (crisner) [reply src/layerData.json]: I got the bounds for the initial map view and added it to the json file for those layers that didn't have bounds. And yes, I manually checked every layer added to the demo page and it works.

Comment (crisner) [reply src/layerData.json]: It didn't occur to me that I could code it to work when there were no bounds given at the time I wrote it. 😅

**Comment by crisner** [comment]
Yes. Good for a merge. Thanks!

File: src/layerData.json (hunk 1 above)
Comment (crisner) [reply src/layerData.json]: I got the bounds for JusticeMap layer by running `map.getBounds()`. Bounds and zoom for other layers were taken from the info.json file. I am not sure how they were established.

**Comment by jywarren** [comment]
Hi @crisner I noticed some possible duplication between https://github.com/publiclab/leaflet-environmental-layers/blob/master/src/layerData.json and https://github.com/publiclab/leaflet-environmental-layers/blob/master/src/info.json -- are these files redundant? Thank you!

**Comment by crisner** [comment]
`info.json` is used to pull in api url and extents data when creating layers. It is used when creating layers such as [Wisconson](https://github.com/publiclab/leaflet-environmental-layers/blob/master/src/wisconsinLayer.js), [ToxicRelease](https://github.com/publiclab/leaflet-environmental-layers/blob/master/src/toxicReleaseLayer.js), and other layers in [layercode.js](https://github.com/publiclab/leaflet-environmental-layers/blob/master/src/layercode.js).

`layerData.json` is used to pull in data for the layer control menu. I created this file to add layer data that I collected and to avoid corrupting the layers in the process of working on the layer control. This reminds me that I should add some documentation for the control.

We could combine them so that the layer control and the layer files that use them can share the same json file and avoid duplication of data. I could take this up if you think it is better to merge them into a single file.

File: src/layerData.json (hunk 1 above)
Comment (sagarpreet-chadha) [reply src/layerData.json]: Yes they were figured the same way you did 😄

**Pull request merged**